tuxemon/states/world/world_menus.py

```diff
@@ -228,10 +228,9 @@ def open_monster_techs() -> None:
             """Show techniques."""
             self.client.pop_state()
             monster = monster_menu.get_selected_item().game_object
-            self.client.push_state(TechniqueMenuState())
-            local_session.player.game_variables[
-                "open_monster_techs"
-            ] = monster.instance_id.hex
+            self.client.push_state(
+                TechniqueMenuState(kwargs={"slug": monster})
```

Comment:
This isn't how you use kwargs.

`TechniqueMenuState(slug=monster)`
`slug = kwargs["slug"]`

And it's not a slug, it's a monster, right?

Reply:
exactly, it's a monster: Monster

Reply:
Fixed